src/pages/Filter/Filter.js

```diff
@@ -1,5 +1,374 @@
+              <SwiperSlide>
+                <ModalInnerText>
+                  <SubTitle>가격 범위</SubTitle>
+                  <PriceInfo> 평균 1박 요금은 ₩140,000입니다</PriceInfo>
+                  <CommonWrapper>
+                    <AllPrice>
+                      <PriceOutline>
+                        <PriceMinMax>최저 요금</PriceMinMax>
+                        <PriceTag>
+                          <WonSymbol>₩</WonSymbol>
+                          <PriceFilter type="number" />
+                        </PriceTag>
+                      </PriceOutline>
+                      <p> - </p>
+                      <PriceOutline>
+                        <PriceMinMax>최고 요금</PriceMinMax>
+                        <PriceTag>
+                          <WonSymbol>₩</WonSymbol>
+                          <PriceFilter type="number" />
+                        </PriceTag>
+                      </PriceOutline>
+                    </AllPrice>
+                  </CommonWrapper>
+                  <CommonWrapper>
+                    <SubTitle>숙소 유형</SubTitle>
+                    <form>
+                      <CheckBoxWrapper>
+                        <CheckBox type="checkbox" />
+                        <div>
+                          <RoomType>집전체</RoomType>
+                          <RoomDescription>
+                            단독으로 사용하는 공간 전체
+                          </RoomDescription>
+                        </div>
+                      </CheckBoxWrapper>
+                      <CheckBoxWrapper>
+                        <CheckBox type="checkbox" />
+                        <div>
+                          <RoomType>개인실</RoomType>
+                          <RoomDescription>
+                            집 또는 호텔의 개인실과 일부 공용 공간
+                          </RoomDescription>
+                        </div>
+                      </CheckBoxWrapper>
+                      <CheckBoxWrapper>
+                        <CheckBox type="checkbox" />
+                        <div>
+                          <RoomType>다인실</RoomType>
+                          <RoomDescription>
+                            다른 사람들과 함께 사용하는 다인실 및 공용 공간{' '}
+                          </RoomDescription>
+                        </div>
+                      </CheckBoxWrapper>
+                    </form>
+                  </CommonWrapper>
+                  <CommonWrapper>
+                    <SubTitle>침실과 침대</SubTitle>
+                    <BedAndBath>침실</BedAndBath>
+                    <ButtonCluster>
+                      <IDCButton>상관없음</IDCButton>
+                      <NumberButton>1</NumberButton>
+                      <NumberButton>2</NumberButton>
+                      <NumberButton>3</NumberButton>
+                      <NumberButton>4</NumberButton>
+                      <NumberButton>5</NumberButton>
+                      <NumberButton>6</NumberButton>
+                      <NumberButton>7</NumberButton>
+                      <NumberButton>8</NumberButton>
```

Comment:
이건 어때요? 
```jsx
Array(8)
  .fill()
  .map((_, i) => i + 1) // 여기까지 Mock data라서 최하단으로 빼서 변수로 관리하면 더 좋음!
  .map((quantity) => <NumberButton>{quantity}</NumberButton>);
```

만약 저라면
ButtonGroup.js 라는 Component를 `components > ButtonGroup` 파일에 만들고

ButtonGroup에는 ButtonGroupBase.js , ButtonGroup.js 를 만들어 따로 관리 할 것 같습니다!

- ButtonGroupBase 에서 나와야 할 UI 예시
<img width="213" alt="스크린샷 2022-11-02 오후 12 19 12" src="https://user-images.githubusercontent.com/53929065/199387539-bb214bb4-ce2e-4a3b-bd6e-d699bde7e419.png">  

- ButtonGroup에서 나와야 할 UI 예시
<img width="619" alt="스크린샷 2022-11-02 오후 12 19 25" src="https://user-images.githubusercontent.com/53929065/199387569-2c4f1c2d-3dbd-4759-a13f-965312774680.png">

최종 리팩토링을 하면 
```suggestion
  <ButtonGroup flatButtonList={Array.fill(8).map((_,i) => i + 1)} />
  
  //ButtonGroup
  const ButtonGroup = (props) => {
    const { flatButtonList } = props;
    // some logics...
    return (
      <>
      {
        flatButtonList.map((button)=> <ButtonGroupBase 넘겨줄프롭들/>)
      }
      </>
    )
  }
```

이런 식으로 될 것 같아요

Reply:
넵 감사합니다 역시 기본 JS 로직이 중요하네요 감사합니다!